1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,44 @@
 // Write your code here
+const items = document.querySelectorAll('img');
+const nextItem = document.querySelector('.next');
+const previousItem = document.querySelector('.previous');
+
+let count = 0;
+
+function showNextItem() {
+  items[count].classList.remove('active');
+  if(count < items.length - 1) {
+    count++;
+  } else {
+    count = 0;
+  }
+  items[count].classList.add('active');
+  }
+
+function showPreviousItem() {
+  items[count].classList.remove('active');
+  if(count > 0) {
+    count--;
+  } else {
+    count = items.length - 1;
```

Comment:
use of `count` variable tells me that it's more like `index`. 
please rename it to `index` or `idx`.